webpack/scenes/ContentViews/Details/Versions/Compare/CVVersionCompareTable.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import {
+  useSelector,
+} from 'react-redux';
+import PropTypes from 'prop-types';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { TableVariant, Tr, Th, Tbody, Td, Thead } from '@patternfly/react-table';
+import { TableType } from './CVVersionCompareConfig';
+import TableWrapper from '../../../../../components/Table/TableWrapper';
+
+const CVVersionCompareTable = ({
+  tableConfig: {
+    name,
+    responseSelector,
+    statusSelector,
+    autocompleteEndpoint,
+    fetchItems,
+    columnHeaders,
+    disableSearch,
+  }, versionOne, versionTwo, currentActiveKey,
+}) => {
+  const [searchQuery, updateSearchQuery] = useState('');
+
+  const response = useSelector(responseSelector);
+  const { results, ...metadata } = response;
+  const status = useSelector(statusSelector);
+  return (
+    <TableWrapper
+      {...{
+        metadata,
+        searchQuery,
+        updateSearchQuery,
+        status,
+        autocompleteEndpoint,
+        disableSearch,
+      }}
+      ouiaId="content-view-version-comparison-table"
+      fetchItems={fetchItems}
+      additionalListeners={[versionOne, versionTwo, currentActiveKey]}
+      emptySearchTitle={__('Your search returned no matching ') + name}
+      emptySearchBody={__('Try changing your search criteria.')}
+      emptyContentTitle={__('No matching ') + name + __(' found.')}
```

Comment:
String concat etc doesn't work in the translator. Let's leep the text simple like "No matching content found". That should be good enough.

Reply:
You can also use `<FormattedMessage>` if you want :yum: